Update LICENSE.md

Dargon789 opened the pull request:
### What changed? Why?


#### Qualified Impact
<!-- Please evaluate what components could be affected and what the impact would be if there was an
error. How would this error be resolved, e.g. rollback a deploy, push a new fix, disable a feature
flag, etc... -->

Newportgogogadit committed:
Update LICENSE.md

Dargon789 committed:
Update package-lock.json

Dargon789 committed:
Add .circleci/config.yml (#6)

snyk-io[bot] committed:
feat: upgrade @types/node from 20.17.19 to 24.5.2 (#5)

Snyk has created this PR to upgrade @types/node from 20.17.19 to 24.5.2.

See this package in npm:
@types/node

See this project in Snyk:
https://app.snyk.io/org/dargon789/project/81aa3d40-9370-47d9-bdcc-5fa21ad09b2e?utm_source=github-cloud-app&utm_medium=referral&page=upgrade-pr

Co-authored-by: snyk-io[bot] <141718529+snyk-io[bot]@users.noreply.github.com>

Dargon789 committed:
Merge branch 'master' of https://github.com/Dargon789/coinbase-sdk-nodejs

Dargon789 committed:
Update package-lock.json

snyk-io[bot] committed:
fix: upgrade viem from 2.21.6 to 2.37.9 (#7)

Snyk has created this PR to upgrade viem from 2.21.6 to 2.37.9.

See this package in npm:
viem

See this project in Snyk:
https://app.snyk.io/org/dargon789/project/d151c5c6-a292-49e3-97b9-f493a6e00b24?utm_source=github-cloud-app&utm_medium=referral&page=upgrade-pr

Co-authored-by: snyk-io[bot] <141718529+snyk-io[bot]@users.noreply.github.com>

snyk-io[bot] committed:
fix: upgrade typescript from 5.4.5 to 5.9.2 (#4)

Snyk has created this PR to upgrade typescript from 5.4.5 to 5.9.2.

See this package in npm:
typescript

See this project in Snyk:
https://app.snyk.io/org/dargon789/project/81aa3d40-9370-47d9-bdcc-5fa21ad09b2e?utm_source=github-cloud-app&utm_medium=referral&page=upgrade-pr

Co-authored-by: snyk-io[bot] <141718529+snyk-io[bot]@users.noreply.github.com>

snyk-io[bot] committed:
feat: upgrade @typescript-eslint/parser from 7.18.0 to 8.44.0 (#3)

Snyk has created this PR to upgrade @typescript-eslint/parser from 7.18.0 to 8.44.0.

See this package in npm:
@typescript-eslint/parser

See this project in Snyk:
https://app.snyk.io/org/dargon789/project/81aa3d40-9370-47d9-bdcc-5fa21ad09b2e?utm_source=github-cloud-app&utm_medium=referral&page=upgrade-pr

Co-authored-by: snyk-io[bot] <141718529+snyk-io[bot]@users.noreply.github.com>

Dargon789 committed:
fix: upgrade viem from 2.21.56 to 2.40.3 (#9)

Snyk has created this PR to upgrade viem from 2.21.56 to 2.40.3.

See this package in npm:
viem

See this project in Snyk:
https://app.snyk.io/org/apichanmittare/project/71ea425b-84e1-4ad4-9f8c-c49393dc125a?utm_source=github&utm_medium=referral&page=upgrade-pr

Co-authored-by: snyk-bot <snyk-bot@snyk.io>

Dargon789 committed:
Merge branch 'master' into pr/430

cb-heimdall commented:
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td><code>2</code> if repo is sensitive</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>
<!-- CODEOWNERS & Bar Raisers -->
<!-- Power Rangers -->
<!-- Links -->

Dargon789 committed:
fix: quickstart-template/webhook/package.json & quickstart-template/webhook/package-lock.json to reduce vulnerabilities (#13)

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-QS-14724253

Co-authored-by: snyk-bot <snyk-bot@snyk.io>

Dargon789 committed:
fix: quickstart-template/discord_tutorial/package.json & quickstart-template/discord_tutorial/package-lock.json to reduce vulnerabilities (#15)

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-AXIOS-15252993

Co-authored-by: snyk-bot <snyk-bot@snyk.io>
